packages/room/peer/peer-types.d.ts

```diff
@@ -54,4 +54,21 @@ export declare namespace RoomPeerType {
   type RTCRtpSVCTransceiverInit = RTCRtpTransceiverInit & {
     sendEncodings?: RTCRtpSVCEncodingParameters[]
   }
+
+  type AudioLevel = {
+    sequence_no: number
+    timestamp: number
+    audio_level: number
+  }
+
+  type VoiceActivity = {
+    type: string
+    track_id: string
+    stream_id: string
+    ssrc: number
+    clock_rate: number
+    audio_levels?: AudioLevel[]
+  }
```

Comment:
Most type objects are pascal case not camel case like [this one](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection). If it is variable then it's a camel case. What do you think?

Reply:
I'm sorry mas @tyohan , what I concerned was not the naming on the type like `VoiceActivity`. But, the object properties such as `audio_level`, `track_id`, `stream_id`, `clock_rate`. Because these properties will be used by developers and the conventional naming case in JavaScript is using the camel case.

So, the usage in the user should be
```js
peer.addEventListener('voiceactivity', (event) => {
  event.detail.trackID // not track_id
  event.detail.streamID // not stream_id
  event.detail.clockRate // not clock_rate
  event.detail.audioLevels // not audio_levels

  // sequence_no => sequenceNumber (I think this is a better name, what do you think mas?)
  // audio_level => audioLevel
})
```

Reply:
Ahhh, got it. Those data are returned from our API, so I need to adjust it directly on our API, no changes is needed on this client SDK.

Reply:
Ah yes, and I'm sorry I forgot the data is from the API. What we can do is probably just as simple as mapping the data returned into a new object similar to this approach

```js
_onVoiceActivity = (vad) => {

      const event = new CustomEvent('voiceactivity', {
        detail: {
          voiceActivity: {
            trackID: vad.track_id,
            streamID: vad.stream_id
            clockRate: vad. clock_rate,
            audioLevels: vad. audio_levels,
            // ... etc
          },
        },
      })
```

Reply:
@faiq-naufal this is updated with the following:

```js
peer.addEventListener('voiceactivity',(ev)=>{
  // type AudioLevel = {
  //   sequenceNo: number
  //   timestamp: number
  //   audioLevel: number
  // }

  // type VoiceActivity = {
  //   type: string
  //   trackID: string
  //   streamID: string
  //   ssrc: number
  //   clockRate: number
  //   audioLevels?: AudioLevel[]
  // }
  console.log(ev.detail.voiceActivity)
})
```